sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Azure.Monitor.OpenTelemetry.AspNetCore.csproj

```diff
@@ -14,10 +14,12 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTelemetry.Extensions.Hosting" />
     <PackageReference Include="System.Text.Encodings.Web" />
     <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" />
     <PackageReference Include="Microsoft.AspNetCore.Http.Features" />
```

Comment:
Do we still need these references `Microsoft.AspNetCore.Http.Abstractions`, `System.Text.Encodings.Web` and `Microsoft.AspNetCore.Http.Features`?

Reply:
I don't know the history of each packagereference. looks like there are some in both projects that could be removed. I'll do that now.